pkgs/development/python-modules/micawber/default.nix

```diff
@@ -1,19 +1,29 @@
-{ lib, buildPythonPackage, fetchPypi, beautifulsoup4 }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, beautifulsoup4
+, pythonOlder
+}:
 
 buildPythonPackage rec {
   pname = "micawber";
-  version = "0.5.4";
+  version = "0.5.5";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "003c5345aafe84f6b60fd289c003e8b1fef04c14e015c2d52d792a6b88135c89";
+    sha256 = "sha256-CsWBS2X/KngfsTlLkaI6ipX3NJK2u49wW67q2C6t1UM=";
   };
 
-  propagatedBuildInputs = [ beautifulsoup4 ];
+  propagatedBuildInputs = [
+    beautifulsoup4
+  ];
 
   meta = with lib; {
     homepage = "https://micawber.readthedocs.io/en/latest/";
-    description = "A small library for extracting rich content from urls";
+    description = "Module for extracting rich content from URLs";
```

Comment:
where does this description come from? the old is used in official places

https://pypi.org/project/micawber/
https://micawber.readthedocs.io/en/latest/
https://github.com/coleifer/micawber

Reply:
> where does this description come from?

I'm tried to "free" descriptions for Python module from "Marketing" terms...if something is "fast", "super optimized", "the only on planet earth" or "big" should be determined by the users not by upstream :wink: